### PR TITLE
fix: delete text and textArranmgnet

### PR DESCRIPTION
**Pull request by nahmed22**
Delete Common Text and TextArrangement  (setting value of Text to None")

### CODE REVIEW

**Review by voicis — CHANGES_REQUESTED** [review]

File: packages/fiori-annotation-api/src/cds/preprocessor.ts [hunk 1]
```diff
@@ -354,6 +400,8 @@ class ChangePreprocessor {
             return createDeleteTargetChange(parentPointer);
         } else if (grandParent.type === ANNOTATION_GROUP_TYPE) {
             return createDeleteAnnotationGroupChange(parentPointer);
+        } else if (grandParent.type === ANNOTATION_GROUP_ITEMS_TYPE) {
+            return createDeleteAnnotationGroupItemsChange(parentPointer.split('/').slice(0, -1).join('/'));
```
[the remark this answers is not in the document]
Comment (vadson71) [reply packages/fiori-annotation-api/src/cds/preprocessor.ts]: done

File: packages/fiori-annotation-api/src/cds/preprocessor.ts [hunk 2]
```diff
@@ -273,12 +296,30 @@ class ChangePreprocessor {
         deletionMap: Record<string, DeletionIndex[]>,
         insertionMap: Record<string, boolean>
     ) {
-        const childPointers = new Set([...parent.items.items.map((_, i) => `${parentPointer}/items/items/${i}`)]);
+        const parentPointerForGroup = parentPointer.split('/').slice(0, -1).join('/');
```
[the remark this answers is not in the document]
Comment (vadson71) [reply packages/fiori-annotation-api/src/cds/preprocessor.ts]: corrected

**Review by voicis — APPROVED** [review]
- Code changes looks good
- Review comments addressed
- Great test coverage

**Review by Jimmy-Joseph19 — APPROVED** [review]
Changes looks - good.
Tested locally.
Test coverage - good.
Approving

**Pull request merged**